src/OrderLinesList/OrderLinesList.js

```diff
@@ -20,7 +20,10 @@ import {
   VENDORS,
 } from '../components/Utils/resources';
 // import OrderLinesFilters from './OrderLinesFilters';
-// import { filterConfig } from './OrdersLinesFilterConfig';
+import {
+  // filterConfig,
```

Comment:
just keep commented line as is, to reduce number of conflicts

Reply:
Got it. Fixed